support/controlplane-component/assets/cluster-autoscaler/serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
For service accounts, we need to make sure we add the corresponding pull secret, as in
https://github.com/openshift/hypershift/blob/ff09b50f5b1c96cb38bae0021d4c74b2ba5fc02d/control-plane-operator/controllers/hostedcontrolplane/autoscaler/reconcile.go#L281

Reply:
this is done generically in `upsert.go` here https://github.com/openshift/hypershift/pull/4708/files#diff-070a5e62112968f41a0e97012266c5e533fd8a78f7c12741a45fcf70867c8535R293